src/cmd/deploy.ts

```diff
@@ -507,20 +546,94 @@ export default class DefenderDeploy {
         retrieveExisting,
         // on update
         async (sentinel: YSentinel, match: DefenderSentinel) => {
+          const isForta = (o: DefenderSentinel): o is DefenderFortaSentinelResponse => o.type === 'FORTA';
+          const isBlock = (o: DefenderSentinel): o is DefenderBlockSentinelResponse => o.type === 'BLOCK';
+
+          // Warn users when they try to change the sentinel network
+          if (match.network !== sentinel.network) {
+            this.log.warn(
+              `Detected a network change from ${match.network} to ${sentinel.network} for Sentinel: ${match.stackResourceId}. Defender does not currently allow updates to the network once a Sentinel is created. This change will be ignored. To enforce this change, remove this sentinel and create a new one. Atlernatively, you can change the unique identifier (stack resource ID), to force a new creation of the sentinel.`,
```

Comment:
Does this apply to all sentinels, or only BLOCK ones?

Reply:
All, should it be only block sentinels?

Reply:
You're right, I just tested and it's not possible to change network in FORTA sentinels. Disregard this comment!